examples/apps/generic_client/README.md

```diff
@@ -49,16 +47,42 @@ worker details of first worker in the list, and submit work order.
 
 If `--uri` is passed, `--mode` is not used. It will fetch the worker details
 from the LMDB database and submit work order to the first available worker.
-The TCF Listener uses TCP port 1947, so the URI is usually
+The TCF Listener uses TCP port 1947, so the default URI is
 `http://localhost:1947` .
 
+## Using the Generic Client
+
+The command line client `generic_client.py` allows you to submit
+worker requests on the command line.
+
+1. If needed, update the Ethereum account and direct registry contract
+   information in `docker/Dockerfile.tcf-dev` and
+   `examples/common/python/connectors/tcf_connector.toml`
+2. Follow the build instructions in the
+   [build document](../../../BUILD.md#dockerbuild)
+3. Activate the Python virtual environment with
+   ```bash
+   source $TCF_HOME/tools/build/_dev/bin/activate
```

Comment:
Is this steps for docker build or standalone build. Python virtual environment is removed for docker build in PR -> https://github.com/hyperledger-labs/trusted-compute-framework/pull/130

Reply:
Why is the Python virtual environment removed in PR #130 ? How will it work without the virtual environment?

Reply:
The answer about removing the Python virtual environment is in PR #142 .
I removed the above lines from this PR (lines 63-66).

Reply:
Please re-review